shared/constants/transaction.js

```diff
@@ -132,3 +144,47 @@ export const TRANSACTION_GROUP_CATEGORIES = {
   SIGNATURE_REQUEST: 'signature-request',
   SWAP: 'swap',
 };
+
+/**
+ * @typedef {Object} TxParams
+ * @property {string} from - The address the transaction is sent from
+ * @property {string} to - The address the transaction is sent to
+ * @property {string} value - The amount of wei, in hexadecimal, to send
+ * @property {number} nonce - The transaction count for the current account/network
+ * @property {string} gasPrice - The amount of gwei, in hexadecimal, per unit of gas
+ * @property {string} gasLimit - The max amount of gwei in, in hexadecimal, the user is willing to pay
```

Comment:
This should be `gas` I think
```suggestion
 * @property {string} gas - The max amount of gwei in, in hexadecimal, the user is willing to pay
```